src/@chakra-ui/components/Heading.ts

```diff
@@ -1,38 +1,44 @@
-import { defineStyle, defineStyleConfig } from "@chakra-ui/react"
+import { defineStyleConfig } from "@chakra-ui/react"
 
 import { headingDefaultTheme } from "./components.utils"
 
-const { sizes: defaultSizes } = headingDefaultTheme
+const HEADING_SIZES = ["3xl", "2xl", "xl", "lg", "md", "sm", "xs"] as const
 
-const lineHeightScale = {
-  "4xl": "6xs",
-  "3xl": ["6xs", null, "5xs"],
-  "2xl": ["4xs", null, "5xs"],
-  xl: ["2xs", null, "4xs"],
-  lg: ["2xs", null, "3xs"],
-  md: "xs",
-  sm: "base",
+type SCALE_VALUE = string | [string, null, null, string]
+
+type SIZE_SCALE = { [key in (typeof HEADING_SIZES)[number]]: SCALE_VALUE }
+
+const lineHeightScale: SIZE_SCALE = {
+  "3xl": ["6xs", null, null, "5xs"],
+  "2xl": "4xs",
+  xl: ["2xs", null, null, "4xs"],
+  lg: "2xs",
+  md: ["xs", null, null, "2xs"],
+  sm: ["base", null, null, "xs"],
   xs: "base",
 }
 
-/*
- * Instead of rewriting the entire sizes object, take the existing value from the
- * default theme and replace the lineHeight values.
- */
-const sizes = Object.entries(defaultSizes || {}).reduceRight(
-  (acc, [key, value]) => {
-    return {
-      ...acc,
-      [key]: defineStyle({
-        ...value,
-        lineHeight: lineHeightScale[key],
-      }),
-    }
-  },
-  {
-    ...defaultSizes,
+const fontSizeScale: SIZE_SCALE = {
+  "3xl": ["5xl", null, null, "6xl"],
```

Comment:
This size is not documented in the figma file, right?
![image](https://github.com/ethereum/ethereum-org-website/assets/468158/89fc4ab3-ba02-4452-8df1-0967143daa00)
We have 7 sizes in the code but 6 in figma. Should we have this size in the code? maybe not. cc @nloureiro

Reply:
Whoops! No that should not be there. I'll have it removed.

Reply:
@pettinarip this is now updated, removing the `3xl` token completely.